src/Microsoft.AspNet.Routing/RouteBuilderExtensions.cs

```diff
@@ -9,8 +9,18 @@
 
 namespace Microsoft.AspNet.Builder
 {
+    /// <summary>
+    /// Provides extension methods for <see cref="IRouteBuilder"/> to add routes.
+    /// </summary>
     public static class RouteBuilderExtensions
     {
+        /// <summary>
+        /// Adds a route to the <see cref="IRouteBuilder"/> with the specified name and template.
+        /// </summary>
+        /// <param name="routeCollectionBuilder">The <see cref="IRouteBuilder"/> to add the route to.</param>
```

Comment:
We should rename this to `routeBuilder` - the type was previously called `IRouteCollectionBuilder`